src/nodes/ordered-list-item.ts

```diff
@@ -17,6 +17,6 @@ export class OrderedListItem implements Node {
   }
 
   static isValidPrefix (str: string) {
-    return isNumeric(str[0]) && str[1] === '.' && str[2] === ''
+    return isNumeric(str[0]) && str[1] === '.' && str[2] === ' '
```

Comment:
建议用 `isBlank`，因为我猜测 `\t` 也是合法的。

```suggestion
    return isNumeric(str[0]) && str[1] === '.' && isBlank(str[2])
```

Reply:
这个修改可能需要交还给您处理了，我看了下代码，quote，ordered list，unordered list 使用的都是 " "，需要统一修改。

Reply:
GPT：实际上，Markdown 对于无序列表的格式是比较宽松的，可以使用破折号后面的空格或者制表符来表示列表项。所以你可以把【- abc】换成【-\tabc】，Markdown 都会正确地将其解析为无序列表的一部分。